src/options.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8"/>
```

Comment:
Remove `/` in `/>` (also in the rest of this file). Self-closing tags are only used in XML and XHTML, but this is HTML5, where it does not serve any purpose.

Reply:
Didn't know that! Thanks!